equistore-core/include/equistore.hpp

```diff
@@ -1630,6 +1630,24 @@ class TensorBlock final {
         return array;
     }
 
+    /// Release the `eqs_block_t` pointer corresponding to this `TensorBlock`.
```

Comment:
this seems to be just moved, can we put it back to create less noise or is this intentionally?

Reply:
This is intentional: this function returns a pointer, and is also susceptible to the same issue. But in this case, this is no longer needed in the public interface (this was here to implement rascaline-torch before we had equistore-torch) and so is now moved to the private section of the class.

Reply:
Ah okay missed that it went to private